### DISCUSSION TITLE
How to see info about indexes?

### DISCUSSION THREAD

**Discussion by alanwilter** [duckdb/duckdb#10595] [Q&A] [answered]
I can't find a way to know which indexes does a table have.
Could someone enlighten me please?

**Comment by carlopi** [comment, marked as answer]
You can try `FROM duckdb_indexes();` metadata function, here on the docs: https://duckdb.org/docs/sql/duckdb_table_functions#duckdb_indexes

**Reply by alanwilter** [reply]
Many thanks!
